src/applications/vaos/services/location/index.js

```diff
@@ -217,7 +217,7 @@ export function getFacilityIdFromLocation(location) {
  * @param {String} id A location's fake FHIR id
  */
 export function getSiteIdFromFakeFHIRId(id) {
-  return parseId(id).substr(0, 3);
+  return id ? parseId(id).substr(0, 3) : null;
```

Comment:
Related to your question about this maybe not being in line with FHIR, because we're parsing these ids, I think we could replace some uses of this function with a function that pulls the id out of the `identifiers` list in a Location resource, then do the slice down to 3 digits, and that would I think be enough to keep ourselves in good shape related to FHIR data.

I think we can leave that to a separate PR, though.